pkg/odo/cli/pipelines/wizard.go

```diff
@@ -0,0 +1,147 @@
+package pipelines
+
+import (
+	"fmt"
+	"net/url"
+	"path/filepath"
+	"strings"
+
+	"github.com/openshift/odo/pkg/log"
+	"github.com/openshift/odo/pkg/odo/cli/pipelines/ui"
+	"github.com/openshift/odo/pkg/odo/cli/pipelines/utility"
+	"github.com/openshift/odo/pkg/odo/genericclioptions"
+	"github.com/openshift/odo/pkg/pipelines"
+	"github.com/openshift/odo/pkg/pipelines/ioutils"
+	"github.com/spf13/cobra"
+
+	ktemplates "k8s.io/kubectl/pkg/util/templates"
+)
+
+const (
+	// WizardRecommendedCommandName the recommended command name
+	WizardRecommendedCommandName = "wizard"
+)
+
+var (
+	WizardExample = ktemplates.Examples(`
+    # Wizard OpenShift pipelines.
+    %[1]s 
+    `)
+
+	WizardLongDesc  = ktemplates.LongDesc(`Wizard GitOps CI/CD Manifest`)
+	WizardShortDesc = `Wizard pipelines with a starter configuration`
+)
+
+// WizardParameters encapsulates the parameters for the odo pipelines init command.
+type WizardParameters struct {
+	*pipelines.BootstrapOptions
+	// generic context options common to all commands
+	*genericclioptions.Context
+}
+
+// NewWizardParameters Wizards a WizardParameters instance.
+func NewWizardParameters() *WizardParameters {
+	return &WizardParameters{
+		BootstrapOptions: &pipelines.BootstrapOptions{
+			InitOptions: &pipelines.InitOptions{},
+		},
+	}
+}
+
+// Complete completes WizardParameters after they've been created.
+//
+// If the prefix provided doesn't have a "-" then one is added, this makes the
+// generated environment names nicer to read.
+func (io *WizardParameters) Complete(name string, cmd *cobra.Command, args []string) error {
+	io.GitOpsRepoURL = ui.EnterGitRepo()
+	option := ui.SelectOptionImageRepository()
+	if option == "Openshift Internal repository" {
+		io.InternalRegistryHostname = ui.EnterInternalRegistry()
+		io.ImageRepo = ui.EnterImageRepoInternalRegistry()
+
+	} else {
+		io.DockerConfigJSONFilename = ui.EnterDockercfg()
+		io.ImageRepo = ui.EnterImageRepoExternalRepository()
+	}
+	io.OutputPath = ui.EnterOutputPath()
+	exists, _ := ioutils.IsExisting(ioutils.NewFilesystem(), filepath.Join(io.OutputPath, "pipelines.yaml"))
+
+	if !exists {
+		io.Overwrite = true
```

Comment:
`io.Overwrite` is true in both cases. We probably need a single if condition here

Reply:
changed the logic